minesweeper.py

```diff
@@ -287,6 +287,17 @@ def quit_game(event=None):
     root.destroy()
 
 
+def highscore_window(event=None):
+    window = tkinter.Toplevel()
+    window.resizable(False, False)
```

Comment:
Can this be deleted, i.e. should the user be able to resize the high score window? If this is deleted, you'll need to change `treeview.pack()` to `treeview.pack(fill='both', expand=True)`.

Reply:
```
<ThePhilgrim> I made it non-resizable intentionally
```